proto/rpc/v1/tx.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/transaction.proto";
+import "rpc/v1/meta.proto";
+
+message TxRequest {
+    // the hash of the transaction
+    bytes hash = 1;
+
+    // if true, return data in binary format
+    // TODO: does this make sense for protobuf?
+    bool binary = 2;
+}
+
+message TxResponse {
+
+    // The actual transaction
+    oneof sttx {
```

Comment:
Not sure what `sttx` means. Would recommend dropping the abbreviation.

Reply:
It's a naming convention I copied from rippled (think it means Serialized Type Tx), though I agree that it is very cryptic. What do you think is a good name for the one of? I will probably expand tx and tx_bytes to transaction and transaction_bytes btw.

Reply:
I could call it serialized_transaction

Reply:
changed to serialized_transaction